variables.tf

```diff
@@ -37,6 +37,17 @@ variable "delete_after" {
 variable "backup_resources" {
   type        = list(string)
   description = "An array of strings that either contain Amazon Resource Names (ARNs) or match patterns of resources to assign to a backup plan"
+  default     = []
```

Comment:
Is there a reason to introduce the default on `backup_resources`? I would think this is the primary thing that you're required to pass to this module, so wondering if this was a mistake or if `selection_tags` now makes this no longer required? Excuse my ignorance of AWS Backup.

Reply:
Otherwise it's required, and when using `selection_tags`, one often does not need to explicitly list any resources. It's more of an either-or, though I believe both can be specified. You also can't pass an empty list to a required variable.

This is the best we can do without custom terraform variable validations.

Reply:
Got it -- makes sense 👍